unstructured/ingest/cli/cmds/weaviate.py

```diff
@@ -0,0 +1,70 @@
+import typing as t
+from dataclasses import dataclass
+
+import click
+
+from unstructured.ingest.cli.interfaces import CliConfig, Dict
+from unstructured.ingest.connector.weaviate import SimpleWeaviateConfig, WeaviateWriteConfig
+
+CMD_NAME = "weaviate"
+
+
+@dataclass
+class WeaviateCliConfig(SimpleWeaviateConfig, CliConfig):
+    @staticmethod
+    def get_cli_options() -> t.List[click.Option]:
+        options = [
+            click.Option(
+                ["--host-url"],
+                required=True,
+                help="Weaviate instance url",
+            ),
+            click.Option(
+                ["--class-name"],
+                default=None,
+                type=str,
+                help="Name of the class to push the records into, e.g: Pdf-elements",
+            ),
+            click.Option(
+                ["--auth-keys"],
+                required=False,
+                type=Dict(),
+                help=(
+                    "String representing a JSON-like dict with key,value containing "
+                    "the required parameters to create an authentication object. "
+                    "The connector resolves th authentication object from the parameters. "
+                    "See https://weaviate.io/developers/weaviate/client-libraries/python_v3"
+                    "#api-key-authentication "
+                    "for more information."
+                ),
+            ),
+        ]
+        return options
+
+
+@dataclass
+class WeaviateCliWriteConfig(WeaviateWriteConfig, CliConfig):
+    batch_size: int
```

Comment:
Don't need to redefine `batch_size` here if you're extending `WeaviateWriteConfig`